### DISCUSSION TITLE
Disliked songs are not skipped  [feasibility]

### DISCUSSION THREAD

**Discussion by peterdk** [librespot-org/librespot#620] [Feature Requests]
I listen to my Spotify's release radar a lot, and always dislike a song or two in a week's list.
When not connected to librespot and just playing in Spotify itself, it auto skips these songs when I play the playlist. When connected to librespot, this is however not skipped. Which is annoying.

Anway, I am a (Android/iOS) developer myself, just started learning Rust a bit. So I would love to contribute by adding this functionality, only I don't know if it's possible to do with current librespot endpoints etc. So can maybe someone with a bit of knowledge of librespot internals comment on that?

**Comment by ashthespy** [comment]
There was some preliminary exploration into figuring out what's needed for like/dislike support, have a look at https://github.com/librespot-org/librespot/issues/344
